src/praisonai/praisonai/cli/configuration/model_resolver.py

```
@@ -0,0 +1,152 @@
+    # Honour OPENAI_MODEL_NAME / MODEL_NAME for backward compatibility before
+    # falling back to credential-based inference.
+    env_model = os.environ.get("MODEL_NAME") or os.environ.get("OPENAI_MODEL_NAME")
+    if env_model:
+        if persist:
+            set_recent_model(env_model)
+        return env_model
+
+    try:
+        from ...llm.env import default_model_for_available_provider
+        model = default_model_for_available_provider()
+    except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)